multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -36,14 +36,31 @@ coreos:
       command: start
       enable: true
       content: |
+{{ if isset .ContainerRuntime "rkt" }}
+        [Unit]
+        Requires=load-rkt-stage1.service
+        After=load-rkt-stage1.service
+{{ end }}
         [Service]
         Environment=KUBELET_VERSION={{.K8sVer}}
         Environment=KUBELET_ACI={{.HyperkubeImageRepo}}
-        Environment="RKT_OPTS=--volume dns,kind=host,source=/etc/resolv.conf --mount volume=dns,target=/etc/resolv.conf"
+        Environment="RKT_OPTS=--volume dns,kind=host,source=/etc/resolv.conf \
+          --mount volume=dns,target=/etc/resolv.conf \
+          --volume rktbin,kind=host,source=/usr/bin/rkt \
+          --mount volume=rktbin,target=/usr/bin/rkt \
```

Comment:
I'm ok with it being a temporary hack. 

Long-term it'd be nice if we handled these ongoing issues regarding a privileged container needing various degrees of access to the host all through rkt. Or, in this case, rkt having a recommended way to do rkt-in-rkt that maybe does this behind the scenes or something?

@aaron any thoughts?